src/software/sensor_fusion/sensor_fusion.h

```diff
@@ -104,6 +104,7 @@ class SensorFusion
     std::optional<Ball> ball;
     Team friendly_team;
     Team enemy_team;
+    std::mutex game_state_mutex;
```

Comment:
shouldn't this be in ThreadedSensorFusion?

Reply:
Oh good point, I think this change may have initially been made before we split `SensorFusion` like that. I actually don't think we need the lock at all anymore. Now that SensorFusion just subscribes to a single buffer of `SensorMsg` we no longer have the case where different update functions can be called at the same time (due to SSL Vision and Refbox sending messages at the same time).

@garethellis0 sanity check me here?

Reply:
removed the mutex

Reply:
Seems sane to me.